### DISCUSSION TITLE
[TOOL] Creating video tutorials for Keira3

### DISCUSSION THREAD

**Discussion by FrancescoBorzi** [azerothcore/azerothcore-wotlk#3905] [Tools]
Now that all major features of [Keira3](https://github.com/azerothcore/Keira3/releases) are available, it would be nice if someone could create videos about it.

Some ideas for videos could be:

- A general introduction to Keira3
- Tutorial: how to create/edit a Creature using Keira3
- Tutorial: how to create/edit a Quest using Keira3
- Tutorial: how to create/edit a Gameobject using Keira3
- Tutorial: how to create/edit an Item using Keira3
- Tutorial: how to work with Gossips using Keira3
- Tutorial: how to work with Conditions using Keira3
- Tutorial: how to work with Loots using Keira3
- Tutorial: how to work with SmartAI using Keira3

any of the above would be nice to have. Also showing the result of the work in-game would be very nice.

I have no experience in making videos and I don't have much time for it, so I'm looking for volunteers that could create some videos about this topic.

I feel like Keira3 has great potential and many people just don't know it yet!

**Comment by FrancescoBorzi** [comment]
We have 1 video (in Spanish) from @pangolp:

https://www.youtube.com/watch?v=qfC5GR3-xMw

I look forward to seeing more videos in different languages!

**Reply by jacpurg** [reply]
This video is no longer available.  I have been searching for solid documentation and tutorials for Keira3 for months and found nothing.  Does anyone have any secret links?  Do any of the private server YouTubers have any interest in creating tutorials for private server tools?  It would be so helpful and probably add interest to whatever private server project YOU'RE working on!
